AlfrescoApp/View Controllers/Accounts View Controllers/AccountsViewController.m

```diff
@@ -566,4 +585,57 @@ - (void)setAccessibilityIdentifiers
     self.tableView.accessibilityIdentifier = KAccountVCTableViewIdentifier;
 }
 
+- (void)goToLoginWithAIMSScreen
+{
+    
+}
+
+#pragma mark - AccountPicker Delegate
+
+- (void)addAccount
+{
+    AppDelegate *appDelegate = (AppDelegate *)[[UIApplication sharedApplication] delegate];
```

Comment:
there are too many references to UIApplication with this component, we need to keep loose coupling between classes. I suggest we use a system of delegate which is set in AppDelegate.